shell/common/shell_unittests.cc

```diff
@@ -4570,8 +4690,15 @@ TEST_F(ShellTest, ShellFlushesPlatformStatesByMain) {
     // The construtor for ViewportMetrics{_, width, _, _, _} (only the 2nd
     // argument matters in this test).
     platform_view->SetViewportMetrics(0, ViewportMetrics{1, 10, 1, 0, 0});
-    shell->AddView(1, ViewportMetrics{1, 30, 1, 0, 0});
+    shell->AddView(1, ViewportMetrics{1, 30, 1, 0, 0},
+                   [](bool added) { ASSERT_TRUE(added); });
     platform_view->SetViewportMetrics(0, ViewportMetrics{1, 20, 1, 0, 0});
+
+    // A view can be added and removed all before the isolate launches.
```

Comment:
Can we move this to a separate test? The feature of adding and removing views with calling back the correct result feels enough for a standalone test.

Reply:
Sure, moved!